package.json

```diff
@@ -70,14 +71,14 @@
     "eslint-plugin-prettier": "^4.0.0",
     "fast-check": "^2.17.0",
     "husky": "^5.2.0",
-    "jest": "^27.1.0",
+    "jest": "^27.4.7",
     "license-checker": "^25.0.1",
     "lint-staged": "^12.0.2",
     "prettier": "2.5.1",
     "rollup": "^2.56.2",
     "rollup-plugin-typescript2": "^0.31.1",
     "testcafe": "^1.15.2",
-    "ts-jest": "^27.0.5",
+    "ts-jest": "^27.1.3",
```

Comment:
This are needed for the current PR to be able to use `mocked` exposed from Jest directly (instead of ts-jest, see https://github.com/inrupt/solid-client-vc-js/pull/231/commits/299d8a063c175201d4985e793d7391a0331ce366). I'll rebase when merging so that this commit is present in the history. I don't think it is necessary to have a dedicated rule for this, but I'll open the discussion with the rest of the team.